python/social_bot/tasks.py

```diff
@@ -745,49 +745,41 @@ def task_specific_observation(self, agent):
                 pose = np.concatenate((pose, obj_pos), axis=0)
 
         agent_pose = np.array(agent.get_pose()).flatten()
-        if self._use_full_states or self._use_egocentric_states:
```

Comment:
option _use_full_states is not used now, so can be removed from GoalTask

Reply:
Good point.  Done.